tests/test_telemetry.py

```diff
@@ -371,6 +370,62 @@ def test_app_dependencies_loaded(self):
             if not seen:
                 raise Exception(dependency + " not recieved in app-dependencies-loaded message")
 
+    def setup_app_dependency_loaded_not_sent_dependency_collection_disabled(self):
+        weblog.get("/enable_product")
+
+    @missing_feature(
+        context.library in ("dotnet", "nodejs", "java", "golang", "python"),
+        reason="Weblog endpoint GET /enable_product is not implemented yet. ",
+    )
+    def test_app_started_product_info(self):
+        """Assert that product information is accurately reported by telemetry"""
+
+        def validator(data):
+            if data["request"]["content"].get("request_type") == "app-started":
+                content = data["request"]["content"]
+                products = content["payload"]["products"]
+                assert (
+                    "appsec" in products
+                ), "Product information is not accurately reported by telemetry on app-started event"
+
+        self.validate_library_telemetry_data(validator)
+
+    @irrelevant(library="php")
```

Comment:
Why is it not relevant for those tracers ?

Reply:
In this situation, `@missing_feature` is the good decorator. `@irrelevant` is when the test is not, and will never be executed.

The practical difference is : 

* `missing_feature` => the test is executed.
   * if it fails, nothing is reported
   * if it succeed, it's reported as a warning as`XPASS` 
* `irrelevant` => the test is not executed (as it'll pass).